test/mitmproxy/test_command.py

```diff
@@ -309,6 +310,19 @@ def empty(self) -> None:
         pass
 
 
+def test_collect_commands():
+    """
+        This tests for the error thrown by hasattr()
+    """
+    with mock.patch("mitmproxy.command.hasattr") as mock_hasattr:
+        mock_hasattr.return_value = False
+        with taddons.context() as tctx:
+            mock_hasattr.side_effect = OSError
+            c = command.CommandManager(tctx.master)
+            a = TDec()
+            c.collect_commands(a)
```

Comment:
Don't you want to assert something? How would we tell if the test actually works?

Reply:
The idea here was to see if the command throws any error or not. If it doesn't then the test is successful, I thought that `assert` will not be useful in this case hence decided Thoughts?